.husky/pre-push.js

```diff
@@ -0,0 +1,12 @@
+const branchName = process.argv[2];
+
+const testRegExp = /^MID-[0-9]+_([a-zA-Z0-9]+-?)+[^-]$/g;
+
+const result = testRegExp.exec(branchName);
+if (result === null) {
+  console.log('Branch name should be of the form: "MID-<ticket number>_<branch-name-text-in-kebab-case>"');
+  console.log('Example: "MID-123_add-this-cool-feature"');
+  console.log(`Provided branch name ${branchName}`);
```

Comment:
Consider similar suggestions like for commit messages

Reply:
same here